tests/resources/VCH-Util.robot

```diff
@@ -187,7 +187,8 @@ Gather Logs From Test Server
     Remove File  vic-admin-cookies
     ${out}=  Run  govc datastore.download %{VCH-NAME}/vmware.log %{VCH-NAME}-vmware.log
     Should Contain  ${out}  OK
-    ${out}=  Run  sshpass -p %{TEST_PASSWORD} scp %{TEST_USERNAME}@%{TEST_URL}:/var/log/vmkernel.log ./vmkernel.log
+    ${out}=  Run  sshpass -p %{TEST_PASSWORD} scp -o StrictHostKeyChecking\=no %{TEST_USERNAME}@%{TEST_URL}:/var/log/vmkernel.log ./vmkernel.log
```

Comment:
if you are using the private key, do you need to use sshpass and the password anymore?